src/mesh/mesh.cpp

```diff
@@ -1042,85 +1013,76 @@ void Mesh::ApplyUserWorkBeforeOutput(ParameterInput *pin) {
 }
 
 //----------------------------------------------------------------------------------------
-// \!fn void Mesh::Initialize(int res_flag, ParameterInput *pin)
+// \!fn void Mesh::Initialize(bool init_problem, ParameterInput *pin)
 // \brief  initialization before the main loop
 
-void Mesh::Initialize(int res_flag, ParameterInput *pin, ApplicationInput *app_in) {
+void Mesh::Initialize(bool init_problem, ParameterInput *pin, ApplicationInput *app_in) {
   Kokkos::Profiling::pushRegion("Mesh::Initialize");
-  bool iflag = true;
-  int inb = nbtotal;
+  bool init_done = true;
+  const int inb = nbtotal;
```

Comment:
I wonder if this line is really even necessary. We could just use nbtotal it's a bit more descriptive and it's used on line 1084 anyway.

Reply:
Good point. If there are no objections I'll add that comment with the next PR tomorrow that changes the buffer calls here.